src/components/homepage/signup/SignUp.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+
+import './SignUp.css';
+
+class SignUp extends Component {
+
+    state = {
+        title: '',
+        Username: '',
+        firstname: '',
+        lastname: '',
+        DOB: '',
```

Comment:
write DOB in full i.e dateOfBirth. When someone checks this they might not know what DOB means.

Reply:
done